cinema/serializers.py

```diff
@@ -18,7 +18,56 @@ def update(self, instance, validated_data):
             "description", instance.description
         )
         instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(required=True, max_length=255)
+    last_name = serializers.CharField(required=True, max_length=255)
+
```

Comment:
https://www.django-rest-framework.org/api-guide/fields/#required
it's the default value, so it's unnecessary